training/controllers/auth.py

```diff
@@ -74,7 +85,7 @@ def sign_in():
             else:
                 return render_template('login.html', form=form, alert_mail=True), 451
         else:
-            return render_template('login.html', form=form, alert=True), 450
+                return render_template('login.html', form=form, alert=True), 450
```

Comment:
Why status code 450 and 451? You should look at: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes and pick the right one. Or let us know want to achieve and we'll discuss it.

Reply:
Of course, at first I was not paying attention to consistency in the status codes, I already changed them.